examples/edge.c

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2018 Uber Technologies, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * Example program that finds the edge between two indexes and prints their
+ * boundary vertices.
+ */
+
+#include <h3/h3api.h>
+#include <stdio.h>
+
+int main(int argc, char *argv[]) {
+    H3Index origin = 0x8a2a1072b59ffffL;
+    H3Index destination = 0x8a2a1072b597fffL;  // north of the origin
+
+    H3Index edge = getH3UnidirectionalEdge(origin, destination);
+    printf("The edge is %llx\n", edge);
+
+    GeoBoundary boundary;
+    getH3UnidirectionalEdgeBoundary(edge, &boundary);
```

Comment:
Possibly just for my info: Is there an issue with uninitialized memory here? `getH3UnidirectionalEdgeBoundary` looks like it will only fill in the vertexes it needs, so it's possible that the rest of the `boundary` struct holds uninitialized memory - is this a problem?

Reply:
I don't believe so, but I'll make sure to run these in Valgrind. I don't think it's an issue because that's the same behavior as h3ToGeoBoundary, where `numVerts` is used to indicate how many vertices are valid.